user-guide/Advanced_Functionality/DataMiner_Systems/BPA_tests/BPA_Cassandra_DB_Size.md

```diff
@@ -8,10 +8,12 @@ Cassandra has some operational limits regarding the size of its files, for examp
 
 The *Cassandra DB Size* BPA test checks the size of the local Cassandra file system against a predefined set of rules. You can find information about this BPA test below.
 
-This BPA test is available by default from DataMiner 10.1.4 onwards.
+This BPA test is available from DataMiner 10.1.4 onwards. However, as of DataMiner 10.4.12/10.5.0<!--RN 40751-->, it is no longer included as a standard BPA test in the *System Center* > *Agents* > *bpa* overview and cannot be executed.
```

Comment:
This "cannot be executed" seems a bit odd. Is the BPA test still available elsewhere from 10.4.12/10.5.0 onwards? If yes, we should clarify where, so users know where to find it if needed, and they can execute it from there. 
Or if it's actually no longer available at all, we'd better simplify this change a bit (and remove update info for 10.4.12/10.5.0 below).
```suggestion
This BPA test is available from DataMiner 10.1.4 up to DataMiner 10.4.11/10.4.0 [CU9].<!--RN 40751-->
```

Reply:
The BPA will no longer be available in any way for customers to run it. It won't be deleted internally, so in the future, if anything changes, it can be added again, but for now, there will be no way for clients to access it (unless they know the BPA exists and specifically ask for it to be included).